tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table_test.py

```diff
@@ -0,0 +1,201 @@
+        for table_name in cls.rule_instance.affected_tables:
+            cls.fq_table_names.append(
+                f'{cls.project_id}.{cls.dataset_id}.{table_name}')
+
+        # call super to set up the client, create datasets, and create
+        # empty test tables
+        # NOTE:  does not create empty sandbox tables.
+        super().setUpClass()
+
+    def setUp(self):
+        """
+        Create common information for tests.
+
+        Creates common expected parameter types from cleaned tables and a common fully qualified (fq)
+        dataset name string used to load the data.
+
+        Creates tables with test data for the rule to run on
+        """
+        fq_dataset_name = self.fq_table_names[0].split('.')
+        self.fq_dataset_name = '.'.join(fq_dataset_name[:-1])
+
+        super().setUp()
+
+        # test data for person table
+        person_data_query = self.jinja_env.from_string("""
+            DROP TABLE IF EXISTS
+              `{{project_id}}.{{dataset_id}}.person`;
+            CREATE TABLE
+              `{{project_id}}.{{dataset_id}}.person` (
+                person_id INT64
+              );
+            INSERT INTO
+              `{{project_id}}.{{dataset_id}}.person` (person_id)
+            VALUES
+              (123),
+              (345),
+              (678),
+              (910)
+            """).render(project_id=self.project_id, dataset_id=self.dataset_id)
+
+        # test data for observation table
+        observation_data_query = self.jinja_env.from_string("""
+            DROP TABLE IF EXISTS
+              `{{project_id}}.{{dataset_id}}.observation`;
+            CREATE TABLE
+              `{{project_id}}.{{dataset_id}}.observation` (
+                observation_id INT64,
+                person_id INT64,
+                value_source_concept_id INT64,
+                value_as_concept_id INT64,
+                observation_source_concept_id INT64
+                );
+            INSERT INTO
+              `{{project_id}}.{{dataset_id}}.observation` (observation_id,
+                person_id,
+                value_source_concept_id,
+                value_as_concept_id,
+                observation_source_concept_id)
+            VALUES
+              (111, 123, 1585266, 0, 1585249),
+              (222, 345, 1585266, 0, 1585249),
+              (333, 678, 1585266, 0, 1585249),
+              (444, 910, 1585266, 0, 1585249),
+              (1122, 123, 1585847, 45878463, 1585845),
+              (3344, 345, 1585847, 45878463, 1585845),
+              (5566, 678, 1585847, 45878463, 1585845),
+              (7788, 910, 1585847, 45878463, 1585845)
+            """).render(project_id=self.project_id, dataset_id=self.dataset_id)
+
+        # test data for concept table
+        concept_data_query = self.jinja_env.from_string("""
+            DROP TABLE IF EXISTS
+              `{{project_id}}.{{dataset_id}}.concept`;
```

Comment:
These are symptomatic of a test setup/initialization/tearDown problem.

Reply:
Do you suggest I take out `DROP TABLE` and just have it be `CREATE TABLE`, since I've added in the `tearDown`?